pkg/cmd/server/logs.go

```diff
@@ -33,22 +41,101 @@ var logsCmd = &cobra.Command{
 			query = "follow=true"
 		}
 
-		ws, res, err := apiclient.GetWebsocketConn("/log/server", &activeProfile, &query)
-		if err != nil {
-			log.Fatal(apiclient.HandleErrorResponse(res, err))
+		switch {
+		case fileFlag:
+			readServerLogFile()
+
+		case activeProfile.Id == "default" && activeProfile.Name == "default":
+
+			ws, res, err := apiclient.GetWebsocketConn("/log/server", &activeProfile, &query)
+			switch {
+			case err != nil:
+				log.Error(apiclient.HandleErrorResponse(res, err))
+				readLogsFilePtr := new(bool)
```

Comment:
`readFromFile := true` and then `&readFromFile` in the `Value()` is more appropraite.